pom.xml

```diff
@@ -880,8 +880,8 @@
     <!-- By default only check remote repositories once per week -->
     <maven.repository.update.freqency>interval:10080</maven.repository.update.freqency>
 
-    <!-- Starting from Jenkins 1.625.x Jenkins supports Java 7 only, hence we may want to bump the dependency in components -->
-    <java.level>6</java.level>
+    <!-- Starting from Jenkins 2.54 Jenkins supports Java 8 only -->
```

Comment:
Well, it also supports Java 11 at the moment. I would rephrase or remove the comment entirely

Reply:
Thanks, I adjusted the comment 👍